### PR TITLE
Removing ow gfx groups Part 1

### PR DESCRIPTION
**Pull request by JaredBrian**

### CODE REVIEW

File: ZeldaFullEditor/Data/OverworldMap.cs [hunk 1]
```diff
@@ -760,17 +792,35 @@ public void LoadPalette()
...
+            else if (this.ParentID >= 0x80)
+            {
+                world = 16;
+            }
+
+            for (int i = 0; i <  8; i++)
+            {
+                if (this.StaticGFX[i] == 0xFF)
+                {
+                    this.StaticGFX[i] = (byte)Constants.OverworldCustomDefualtTileGFX[i + world];
```
Comment (scawful) [inline ZeldaFullEditor/Data/OverworldMap.cs:831]: typo: OverworldCustomDefualtTileGFX

File: ZeldaFullEditor/Data/OverworldMap.cs [hunk 2]
```diff
@@ -760,17 +792,35 @@ public void LoadPalette()
...
+            // If the GFX are 0xFF they need to show the defualt GFX instead.
+            int world = 0;
+            if (this.ParentID >= 0x40)
+            {
+                world = 8;
+            }
+            else if (this.ParentID >= 0x80)
+            {
+                world = 16;
+            }
```
Comment (scawful) [inline ZeldaFullEditor/Data/OverworldMap.cs:825]: This handles special worlds too?

**Review by scawful — APPROVED** [review]

File: ZeldaFullEditor/Data/OverworldMap.cs (hunk 2 above)
Comment (JaredBrian) [reply ZeldaFullEditor/Data/OverworldMap.cs:825]: yup!

File: ZeldaFullEditor/Data/OverworldMap.cs (hunk 1 above)
Comment (JaredBrian) [reply ZeldaFullEditor/Data/OverworldMap.cs:831]: very good catch

**Review by Zarby89 — APPROVED** [review]

**Pull request merged**